src/app/demo/demo/demo.component.html

```diff
@@ -745,6 +745,25 @@ <h3 class="dp-options-section">Config options</h3>
         </div>
       </div>
 
+      <div class="dp-option" *ngIf="isValidConfig('apiControlOnly')">
+        <span class="dp-option-header">
+          Show picker via api control only (apiControlOnly):
```

Comment:
After changing the configuration option, please don't forget to change the demo as well